luigi/task.py

```diff
@@ -600,6 +600,25 @@ def output(self):
         """
         return []  # default impl
 
+    def process_requires(self, requires):
+        """
+        Override in "template" tasks which themselves are supposed to be
+        subclassed and thus have their requires() overridden (name preserved to
+        provide consistent end-user experience), yet need to introduce
+        custom behaviour into dependencies.
+
+        ex:
+            class TemplateTask(luigi.Task):
+                log_level = DEBUG
+
+                def process_requires(self, requirements):
+                    for(task in flatten(requirements)):
+                        task.log_level = self.log_level # Propagate log level up the dependency chain.
```

Comment:
There is a Task cache so I would be careful aboud doing mutations. I also said this in previous PR I think. :)